src/main/java/seedu/address/model/transaction/Transaction.java

```diff
@@ -23,20 +35,30 @@ public Transaction(Person payer, Double amount, String description, UniquePerson
 
     public Transaction(Person payer, Double amount, String description, Set<Person> payeesToAdd) {
         UniquePersonList payees = new UniquePersonList();
-        for(Person p: payeesToAdd) {
+        for (Person p: payeesToAdd) {
             try {
                 payees.add(p);
             } catch (DuplicatePersonException e) {
                 System.out.println("Duplicate person" + p.getName() + " not added to list of payees");
             }
         }
 
+        this.id = 0; // TODO: Ensure that this increments by 1 for each new transaction
```

Comment:
Just add in the class : 
private static int lastTransactionId = 0;

and in the constructor : 
this.id = lastTransactionId;
lastTransactionId++;